config/prometheus/monitor.yaml

```diff
@@ -5,12 +5,13 @@ kind: ServiceMonitor
 metadata:
   labels:
     control-plane: controller-manager
-  name: controller-manager-metrics-monitor
-  namespace: system
+  name: cluster-logging-operator-metrics-monitor
+  namespace: openshift-logging
 spec:
   endpoints:
     - path: /metrics
-      port: https
+      targetPort: 8686
+      port: 8686
   selector:
     matchLabels:
-      control-plane: controller-manager
+      control-plane: cluster-logging-operator
```

Comment:
does this match a label set on our operator?

Reply:
yes, label the same as for operator pod

- service 
![screenshot-console-openshift-console apps vparfonov-log1975x devcluster openshift com-2021 12 02-22_40_04](https://user-images.githubusercontent.com/1636592/144499885-e8349af0-d708-4a6b-af8e-29a119a5eba8.png)

- pod 
![screenshot-console-openshift-console apps vparfonov-log1975x devcluster openshift com-2021 12 02-22_40_56](https://user-images.githubusercontent.com/1636592/144499840-a04de3d1-70ac-4bca-afaf-b121be712273.png)